assignments/objects.js

```diff
@@ -19,6 +19,35 @@ const example = {
 
 // Write your intern objects here:
 
+const interns = [
+  {"id": 1,
+  "email": "mmelloy0@psu.edu",
+  "first_name": "Mitzi",
+  "gender": "F"
+},
+{"id": 2,
+  "email": "kdiben1@tinypic.com",
+  "first_name": "Kennan",
+  "gender": "M",
+  "speak": function(){ return 'Hello my name is Kennan!';}
```

Comment:
nice job here creating your objects. You'll be exposed to objects pretty much your entire time programming so it is important to learn to manipulate and work with them